Missionframework/modules/00_init/fnc/fn_init_saveData.sqf

```diff
@@ -14,16 +14,19 @@
     NONE
 
     Returns:
-    ARRAY
+    BOOL
 */
 
 if (KPLIB_param_debugSave > 0) then {diag_log "[KP LIBERATION] [SAVE] Init module saving...";};
 
-private _moduleData = [
-    date, // Current date and time
-    KPLIB_sectors_lockedVeh, // Locked Vehicles array
-    KPLIB_sectors_blufor, // Blufor sectors
-    KPLIB_sectors_fobs apply {getMarkerPos _x} // FOB positions
+// Push module data to the save data array with module name prefix
+KPLIB_save_data pushBack ["init",
+    [
+        date, // Current date and time
+        KPLIB_sectors_lockedVeh, // Locked Vehicles array
+        KPLIB_sectors_blufor, // Blufor sectors
+        KPLIB_sectors_fobs apply {getMarkerPos _x} // FOB positions
+    ]
```

Comment:
Same thing as with loading, public API should be done via method so input can be validated, for eg. `[<STRING>, <ARRAY>] call KPLIB_fnc_init_setSaveData;`.

Reply:
Same as above ^^